src/main/groovy/edu/oregonstate/mist/personsapi/PersonsResource.groovy

```diff
@@ -80,8 +81,9 @@ class PersonsResource extends Resource {
     @GET
     @Path('{osuID: [0-9]{9}}/jobs')
     Response getJobsById(@PathParam('osuID') String osuID) {
-        def jobs = personsDAO.getJobsById(osuID)
-        if (jobs) {
+
+        if (personsDAO.getPersonById(osuID)) {
```

Comment:
Same comment as the one for id image endpoint: This should be doing a more simple query to see if someone exists.